pom.xml

```diff
@@ -224,6 +224,18 @@
                 <artifactId>nimbus-jose-jwt</artifactId>
                 <version>${nimbusds.version}</version>
             </dependency>
+            <dependency>
+                <groupId>org.wso2.carbon.identity.inbound.auth.oauth2</groupId>
+                <artifactId>org.wso2.carbon.identity.oauth</artifactId>
+                <version>${carbon.identity.oauth.version}</version>
+                <scope>compile</scope>
```

Comment:
No need to explicitly mention the compile scope https://maven.apache.org/guides/introduction/introduction-to-dependency-mechanism.html#dependency-scope